osf/models/base.py

```diff
@@ -116,14 +117,38 @@ def clone(self):
 
     def save(self, *args, **kwargs):
         # Make Django validate on save (like modm)
-        if not kwargs.get('force_insert') and not kwargs.get('force_update'):
+        if kwargs.pop('clean', True) and not (kwargs.get('force_insert') or kwargs.get('force_update')):
             try:
                 self.full_clean()
             except DjangoValidationError as err:
                 raise ValidationError(*err.args)
         return super(BaseModel, self).save(*args, **kwargs)
 
 
+class DegenericQuerySet(models.QuerySet):
+    def degeneric(self, **kwargs):
```

Comment:
This doesn't appear to be used anywhere. Remove?

Reply:
Ah, sorry. Left over from search stuff.